docker/dev/centos/README.md

```diff
@@ -0,0 +1,46 @@
+# How to use this Dockerfile
```

Comment:
As the documentation for the CentOS and Ubuntu Dockers is similar I would suggest to move this README to docker/dev and make the content generic i.e. not referencing any specific OS